crates/cli/src/main.rs

```diff
@@ -41,7 +41,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     let cache_dir = config.pantry_dir.parent().unwrap();
     std::fs::create_dir_all(cache_dir)?;
-    let mut conn = Connection::open(cache_dir.join("pantry.db"))?;
+    let mut conn = Connection::open(cache_dir.join("pantry.2.db"))?;
```

Comment:
i sense migrations coming. there's a fair few tools in rust for that. and a few in pkgx, like `goose`.

Reply:
we would certainly be more proper to be able to update the db incrementally and with migrations. That’s work I am not interested in RN tho for sure.